apps/estuary/lib/estuary_web/templates/layout/root.html.eex

```diff
@@ -0,0 +1,19 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="utf-8"/>
+    <meta http-equiv="X-UA-Compatible" content="IE=edge"/>
+    <meta name="viewport" content="width=device-width, initial-scale=1.0"/>
+    <%= csrf_meta_tag() %>
+    <title>Estuary · Data Curator Interface</title>
+    <link rel="stylesheet" href="<%= Routes.static_path(@conn, "/css/app.css") %>"/>
+  </head>
+  <body>
+    <div class="root">
+      <main role="main" class="root__main">
+        <%= render @view_module, @view_template, assigns %>
```

Comment:
When you merge in Cam's stuff, this will need to change to `<%= @inner_content %>`
```suggestion
       <%= @inner_content %>
```